.github/workflows/main.yml

```
@@ -393,20 +393,29 @@ jobs:
       - name: run test suite (Git WD, standard)
         run: |
           bin/ch-test all
+          if [[ $(cat /tmp/ch-test.tmp.$USER/rootemu) != "yes" ]]; then
+            false
+          fi
```

You don't actually need an `if` for this:

```suggestion
          [[ $(cat /tmp/ch-test.tmp.$USER/rootemu) == "yes" ]]
```